js_modules/dagster-ui/packages/ui-core/src/app/useVisibleFeatureFlagRows.oss.tsx

```diff
@@ -27,4 +27,8 @@ export const useVisibleFeatureFlagRows = (): FeatureFlagRow[] => [
     key: 'Disable automatically loading default config in launchpad',
     flagType: FeatureFlag.flagDisableAutoLoadDefaults,
   },
+  {
+    key: 'Revert to separate asset graph groups per code location',
```

Comment:
Maybe "Show separate..." since users might not have a sense of a "revert" here. I'm also a little confused by the language -- is the thing we're aiming to ship represented by the flag-on state or the flag-off state?